app/smc/pytest/e2e_smoke.py

```diff
@@ -14,6 +16,8 @@
 
 logger = logging.getLogger(__name__)
 
+SCRIPT_DIR = pathlib.Path(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
I think this would work too. Not blocking again.
```suggestion
SCRIPT_DIR = pathlib.Path(__file__).parent()
```